include/oneapi/tbb/parallel_reduce.h

```diff
@@ -514,7 +515,7 @@ Value parallel_reduce( const Range& range, const Value& identity, const RealBody
     lambda_reduce_body<Range,Value,RealBody,Reduction> body(identity, real_body, reduction);
     start_reduce<Range,lambda_reduce_body<Range,Value,RealBody,Reduction>,const __TBB_DEFAULT_PARTITIONER>
                           ::run(range, body, __TBB_DEFAULT_PARTITIONER() );
-    return body.result();
+    return std::move(body.result());
```

Comment:
I think this `std::move` does nothing since `body.result()` is an rvalue?

Reply:
Changed to `std::move(body).result()`, as discussed in the previous thread.